driver-core/pom.xml

```diff
@@ -228,7 +228,7 @@
                             <manifestLocation>${project.build.outputDirectory}/META-INF</manifestLocation>
                             <instructions>
                                 <!-- JNR does not provide OSGi bundles, so exclude it; the driver can live without it -->
-                                <Import-Package><![CDATA[com.google.common.*;version="[16.0.1,22)",!jnr.*,io.netty.channel.epoll,*]]></Import-Package>
+                                <Import-Package><![CDATA[com.google.common.*;version="16.0.1",!jnr.*,io.netty.channel.epoll,*]]></Import-Package>
```

Comment:
Why did you force the guava version to 16.0.1? Shouldn't this be `[16.0.1,27)`? (Same happens in driver-extras and driver-mapping).

Reply:
It's not clear, but when you set a specific version it implies "this version or later", from [section 3.2.6 of the OSGi core specification](https://osgi.org/specification/osgi.core/7.0.0/framework.module.html#i3189032):

> If a version range is specified as a single version, it must be interpreted as the range [version,∞). The default for a non-specified version range is 0, which maps to [0.0.0,∞).

Reply:
Oh TIL :) Thank you for enlightening me. Agreed that it is much more flexible to specify a single version then.